src/webrequest.js

```diff
@@ -340,18 +340,22 @@ function getSocialWidgetBlockList(tabId) {
   // a mapping of individual SocialWidget objects to boolean values
   // saying if the content script should replace that tracker's buttons
   var socialWidgetsToReplace = {};
+  green_domains = {}
```

Comment:
What do you guys think about including and documenting a lint tool (maybe [ESLint](https://github.com/eslint/eslint)) config? You could enforce linting as part of the workflow, but even just having shared lint settings is very helpful for getting everyone on the same page code style-wise.